.github/workflows/sync-malicious-packages.yaml

```
@@ -3,7 +3,7 @@ name: Synchronize malicious packages
 on:
   workflow_dispatch: {} # allows manual trigger
   schedule:
-  - cron: '0 0 * * *' # every day at midnight UTC
+  - cron: '0 8 * * 1' # every day at midnight UTC
```

what about on weekdays?
```suggestion
  - cron: '0 8 * * 1-5' # every day at midnight UTC
```